src/geovista/common.py

```diff
@@ -599,11 +603,12 @@ def point_cloud(mesh: pv.PolyData) -> bool:
     .. versionadded:: 0.2.0
 
     """
-    return (mesh.n_points == mesh.n_cells) and (mesh.n_lines == 0)
+    result: bool = (mesh.n_points == mesh.n_cells) and (mesh.n_lines == 0)
+    return result
 
 
 def sanitize_data(
-    *meshes: Iterable[pv.PolyData],
+    *meshes: pv.PolyData,
```

Comment:
I looked this up - the `Iterable` is implicit with `*` parameters, which explains why MyPy was unhappy here.